.github/workflows/gg-shield-action.yaml

```diff
@@ -0,0 +1,38 @@
+name: GitGuardian scan
+
+on: 
+  push:
+  pull_request:
+    # Publish `main` as Docker `latest` image.
+    branches:
+      - main
+      - haruharu720test
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+jobs:
+  scanning:
+    name: GitGuardian scan
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.4.0
+        with:
+          fetch-depth: 0 # fetch all history so multiple commits can be scanned
+        env:
+          GITHUB_TOKEN: "87028c11-8bef-4bd6-b440-5d3851314ead"
+      # - name: Login to DockerHub Registry
+      #   run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+      - name: GitGuardian scan
+        uses: GitGuardian/ggshield-action@master
+        with:
+          args: --show-secrets --exit-zero --all-policies --verbose
+        env:
+          GITHUB_PUSH_BEFORE_SHA: ${{ github.event.before }}
+          GITHUB_PUSH_BASE_SHA: ${{ github.event.base }}
+          GITHUB_PULL_BASE_SHA: ${{ github.event.pull_request.base.sha }}
+          GITHUB_DEFAULT_BRANCH: haruharu720test
+          GITGUARDIAN_API_KEY: bBae0Daed3c0dFD1dC624380aA3FEbBfddD06dCeB3Ca5c2dCADF661C42617Dc65298FcE
+          GITGUARDIAN_INSTANCE: 'https://api.gitguardian.com/'
```

Comment:
@ruzell22 Please double check if we need this (e.g. remove them and then see if the checking process still works or not). If it does work without these then they should be removed. If it does NOT work without these then we MUST:
1. Invalidate this gitguardian API key before someone maliciously uses it
2. Generate a new git guardian API key, send that to me in private and I'll make sure to add it as a git repo level environment secret.